kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/denali.java

```diff
@@ -140,30 +140,18 @@ public void onFailure(Throwable caught) {
 
                                     @Override
                                     public void onSuccess(ArrayList<GwtGroupedNVPair> pairs) {
-                                        denali.this.m_connected = false;
+                                        denali.this.connected = false;
                                         int connectionNameIndex = 0;
                                         for (GwtGroupedNVPair result : pairs) {
```

Comment:
Since you need and increment the index in the for loop I suggest using the for loop with the index: ``for(int connectionNameIndex...``